src/components/WelcomeIntro.js

```diff
@@ -20,10 +26,37 @@ const WelcomeIntro = () => {
     "Magic Beans": 0,
   };
 
-  for (const cabinPoint of data) {
-    const cabin = cabinPoint.fields["Cabins"];
-    cabinPoints[cabin] += 1;
-  }
+  var base = new Airtable({
```

Comment:
i couldn't figure out how to get this working thru curl so i just kinda threw this in here